### ISSUE TITLE
2019_agsec3a household count max values are greater than what people were allowed to report

### ISSUE THREAD

**Issue by KieranCDouglas** [AIDELabAZ/weather_and_agriculture#18]
**Describe the bug**
Working in 2019_agsec3a and noticed that the household count values seemed odd. There are a ton of 0s and the max values exceed the 1-5 that people were allowed to report. This is in section 5 Labor.

![Uploading Screenshot 2024-04-19 at 12.18.43 PM.png…]()

**Comment by aljosephson** [comment]
Okay! So, the variables which were previously giving household counts, in this round, give a count of labor days. So, the variable which you were using was counting, not people, but days - hence the large numbers.

I looked into it and even more strangely, there was NO household labor. Weird. Turns out, that was in a difference file `agsec3a_1`. I wrote code that collapses that separate file (lines 39 through 54) and then integrates it back in when dealing with labor in the `agsec3a` file (lines 183 through 185).

Strange round to round stuff - but always be a lookout for what the question is asking, even if it is different than previous rounds. I think this file should be set now, but go ahead and take a look to confirm @KieranCDouglas.
